_infra/helm/locust/values.yaml

```diff
@@ -21,6 +21,7 @@ locust:
       test_respondents: 10
       GOOGLE_CLOUD_PROJECT: "ras-rm-performance-20220908"
       GCS_BUCKET_NAME: "ras-rm-performance-20220908-locust"
+      CSRF_ENABLED: true
```

Comment:
I have given up with this in spinnaker, any changes in yaml values unless you jump through a load of hoops are unreliable until merge. So this can be tested I have removed the if statement in the code, but left the var in so I can loop back around after merge and put it back in.